src/main/java/io/vertx/core/http/HttpClient.java

```diff
@@ -1089,6 +1369,31 @@ WebSocketStream websocketStream(String requestURI, MultiMap headers, WebsocketVe
                                   String subProtocols);
 
   /**
+   * Set a redirect handler for the web client.
```

Comment:
for the "http client" would be better to avoid confusion with the vertx-web client